### ISSUE TITLE
CinnVIIStarkMenu@NikoKrause - applications can be clicked and launched outside of menu's window.

### ISSUE THREAD

**Issue by junglized** [linuxmint/cinnamon-spices-applets#5022]
```
 * Applet version/Build date (to be found in "About.." or in "metadata.json")
          Version 1.32 (2023-07-08 22:25:36)
 * Cinnamon version (cinnamon --version)
           Cinnamon 5.8.4
 * Distribution - (Mint 17.2, Arch, Fedora 25, etc...)
          Linux Mint 21.2
 * Graphics hardware *and* driver used
          GeForce GTX 1050 Ti, nvidia-driver-535
 * Snippet or attachment of ~/.cinnamon/glass.log
 * Snippet or attachment of ~/.xsession-errors

 ```

**Notify author of applet**
@NikoKrause

**Issue**
When the menu is opened from the panel sitting at the top, list of applications extend below menu window thus making those apps clickable.

**Steps to reproduce**
Open menu, move mouse cursor below menu window, see applications tooltips being displayed, list of apps can still be scrolled up and down.

**Expected behaviour**
Aplications should not be clickable outside of menu's window.

**Other information**
[Peek 2023-09-02 19-48.webm](https://github.com/linuxmint/cinnamon-spices-applets/assets/26525585/06f0ab44-fa21-42ed-ae84-fd84e1ae42fd)

**Comment by fredcw** [comment]
Duplicate of https://github.com/linuxmint/cinnamon/issues/11123

**Comment by rcalixte** [comment]
Fixed in #6221
